static/js/components/CommentAttachmentPreview.jsx

```diff
@@ -149,7 +162,9 @@ const CommentAttachmentPreview = ({
 
   let jsonFile = {};
   try {
-    jsonFile = isCached ? JSON.parse(commentAttachment.attachment) : {};
+    jsonFile = isCached
+      ? JSON.parse(Object.entries(commentAttachment)[1][1])
```

Comment:
What is this [1][1] thing?

Reply:
Object.entries(commentAttachment) is an array of arrays as you can see below
![image](https://user-images.githubusercontent.com/60301767/163385001-8e488ae3-ef04-43a5-aa76-1b1ace0ee6f0.png)

and if you access the first array at index 1, you will access the attachment array and if you access this array at index 1, you will have the content of your json file (as this occur only when dealing with json files)